chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -38,6 +36,8 @@ use tokio_util::sync::PollSemaphore;
 use crate::peer::peer_actor::PeerActor;
 use crate::peer_manager::peer_store::{PeerStore, TrustLevel};
 use crate::routing::codec::Codec;
+use crate::stats::metrics;
+use crate::stats::metrics::NetworkMetrics;
```

Comment:
@matklad - as this is rust code style question.

But from what I've seen in Zulip, we should aim at having 'one single block of imports' and let cargo fmt handle it

Reply:
>we should aim at having 'one single block of imports' and let cargo fmt handle it

Not so sure -- I know that at least nodeX team expects several blocks of imports (cc @khorolets), as in the title of the PR. 

I think the process here should be:

* take a look at what our current practices are, and pick some rule which is the closest to them
* submit a PR with `docs/style.md` document that documents those practices
* start aligning the code. 

Let me actually do that...

Reply:
https://github.com/near/nearcore/pull/5262 -- didn't go as far as figuring out whether we need import grouping, but that's a start.

Reply:
@matklad I have a mixed feeling about this. I would rather put all imports together and let cargofmt figure it out. Unfortunately, splitting imports into 4 groups adds a bit of extra maintenance, that's not really bringing us any value.
The only reason why I reformatted network code to be like this, was because, I saw we had a code with imports at the beggining (std, empty line, libraries, empty line, near, empty like create) and then some random imports at the end.

My proposal would be to keep a single block of imports and let rustfmt reformat it.

The reason is that, rustfmt will already split imports in 4 groups.
- packages with `crate::` will be on top
- packages that start with `near-` will be together
- packages with std will be kept together

Reply:
Yeah, at this point I am leaning towards not grouping the imports, as that's easier to make consistent.

Reply:
There are some bad things about `cargo fmt` is that it doesn't separate libraries from `std` or any other standard packages from imported packages from github.